UnitsNet/UnitDefinitions/ElectricChargeDensity.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "ElectricChargeDensity",
+  "BaseUnit": "CoulombPerMeterCube",
+  "XmlDoc": "In electromagnetism, charge density is a measure of the amount of electric charge per unit length, surface area, or volume.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Charge_density",
+  "Units": [
+    {
+      "SingularName": "CoulombPerMeterCube",
```

Comment:
Please use `CubicMeter` to be consistent with `Volume`

Reply:
Good point, we will modify as requested.